support/main/default/classes/TriggerControlTest.cls

```diff
@@ -1,137 +1,243 @@
+        trigObj.enabled = enabled;
+        return trigObj;
+    }
+
+    // Add additional params for each trigger evt?
+    private static TriggerConfigService.HandlerConfig generateHandlerTestData(String className, Boolean enabled) {
+        TriggerConfigService.HandlerConfig handler = new TriggerConfigService.HandlerConfig();
+        handler.className = className;
+        handler.enabled = enabled;
+        handler.autoBind = true;
+        handler.order = 0;
+        handler.beforeInsert = true;
+        handler.afterInsert = false;
+        handler.beforeDelete = true;
+        handler.afterDelete = false;
+        handler.afterUndelete = false;
+        handler.beforeUpdate = true;
+        handler.afterUpdate = false;
+        return handler;
+    }
+
+    private static TriggerConfigService.ObjectConfig addChildHandlerToObj(
+        TriggerConfigService.ObjectConfig obj,
+        TriggerConfigService.HandlerConfig handler
+    ) {
+        if (obj.handlers == null) {
+            obj.handlers = new Map<String, TriggerConfigService.HandlerConfig>();
+        }
+        obj.handlers.put(handler.className, handler);
+        obj.handlerOrder.add(handler.className);
+        if (!handler.enabled) {
+            obj.disabledHandlers.add(handler.className);
+        }
+        return obj;
+    }
+
+    private static Map<String, TriggerConfigService.ObjectConfig> disableObj(
+        Map<String, TriggerConfigService.ObjectConfig> mdt,
+        String objectName
+    ) {
+        TriggerConfigService.ObjectConfig obj = mdt.get(objectName);
+        obj.enabled = false;
+        mdt.put(objectName, obj);
+        return mdt;
+    }
+
+    private static Map<String, TriggerConfigService.ObjectConfig> disableHandler(
+        Map<String, TriggerConfigService.ObjectConfig> mdt,
+        String objectName,
+        String className
+    ) {
+        TriggerConfigService.ObjectConfig obj = mdt.get(objectName);
+        obj.handlers.get(className).enabled = false;
+        obj.disabledHandlers.add(className);
+        mdt.put(objectName, obj);
+        return mdt;
+    }
+
+    private static void checkContact(Id contactId, Boolean handler1Fired) {
+        Contact contact = [SELECT Handler1__c FROM Contact WHERE Id = :contactId];
+        System.assertEquals(contact.Handler1__c, handler1Fired);
+    }
+
+    private static void checkAccount(Id acctId, Boolean handler1Fired, Boolean handler2Fired) {
+        Account acct = [SELECT Handler1__c, Handler2__c FROM Account WHERE Id = :acctId];
+        System.assertEquals(acct.Handler1__c, handler1Fired);
+        System.assertEquals(acct.Handler2__c, handler2Fired);
+    }
+
+    private static Id createAccount() {
+        Account record = new Account(Name = 'Test');
+        insert record;
+        return record.Id;
+    }
+
+    private static Id createContact() {
+        Contact record = new Contact(LastName = 'Test');
+        insert record;
+        return record.Id;
+    }
```

Comment:
great test helper methods

Reply:
Pretty sure this was your code lol - nice work

Reply:
was referring to your continuation of my nice work :)